features/testbot/weight.feature

```diff
@@ -52,11 +52,11 @@ Feature: Weight tests
             | abc   |
 
         When I route I should get
-            | waypoints | route   | distances | weights | times   | a:distance                | a:duration | a:weight | a:speed |
```

Comment:
This seems to be caused by the fact that our JSON renderer truncates numbers to have maximum 10 digits after dot https://github.com/Project-OSRM/osrm-backend/blob/9ad432c5b25430cac4140288a242dcd78ad00724/include/util/json_renderer.hpp#L60